sec-keys/openpgp-keys-emacs/openpgp-keys-emacs-20250414.ebuild

```diff
@@ -0,0 +1,19 @@
+# Copyright 1999-2025 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=8
+
+DESCRIPTION="Stefan Kangas' OpenPHP keys used for signing GNU Emacs releases"
```

Comment:
PGP, not PHP.

Also, it may not (only) be Stefan Kangas in the future, or we may want to retroactively add keys of other Emacs maintainers. So I'd rather avoid the name in the description.